build.gradle.kts

```diff
@@ -65,7 +77,7 @@ jsonSchema2Pojo {
 }
 
 
-val jacocoExclude = listOf("**/generated/**", "**/api**", "**/request/**")
+val jacocoExclude = listOf("**/generated/**", "**/enums/**", "**/api**", "**/request/**")
```

Comment:
I think we need a comment here - api and request sound like the same thing to me (and aren't they all generated?) would be good to document the intended classifications

Reply:
commented together with @starswan